.eslintrc.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
nit: thoughts on just using something like https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb here? It would help reduce some of the churn and combined with `--fix` would hopefully help automate most things.